src/util/pointer_expr.h

```diff
@@ -386,7 +386,7 @@ class pointer_in_range_exprt : public ternary_exprt
   }
 
   // translate into equivalent conjunction
-  exprt lower() const;
+  exprt lower(const namespacet &ns) const;
```

Comment:
do we have a trait or abstract interface for expressions that can be lowered or is this method added on a per-expression basis